src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Trace.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Types.Enums;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal sealed class TraceFunction : BuiltinFunction
```

Comment:
>TraceFunction [](http://example.com/codeflow?start=26&length=13)

Since this function comes from Power Apps, we need to remove it from that product and use this one.

Reply:
Agree, I will have a follow-up PR for absorbing these changes.